packages/phone-number-privacy/signer/scripts/createBlsKeyPair.ts

```diff
@@ -0,0 +1,15 @@
+// tslint:disable: no-console
```

Comment:
it appears like a lot of files were duplicated in both directories.  can't they just reference some shared files? this seems like a maintainability nightmare

in some cases I don't think the combiner needs all these files, or it just needs parts of it (though it would be tedious and harder to reverse to separate logic needed from each service so I feel a shared file would be better).

Reply:
Yep I intend to clean things up as we chisel down the functionality in each package to what it actually needs. I did a dumb copy here intentionally to keep this large PR easy to review :)

Reply:
gotcha -- step one of many then :)